Assets/Scripts/System/GameSync.cs

```diff
@@ -350,9 +351,13 @@ private class Game : SluggedItem {
             installationMetadata.Add("min_players", new JSONData(minPlayers));
             installationMetadata.Add("max_players", new JSONData(maxPlayers));
             installationMetadata.Add("executable", new JSONData(executable));
-            installationMetadata.Add("legacy_controls", new JSONData(legacyControls));
             installationMetadata.Add("image_url", new JSONData(imageURL));
 
+
+            JSONClass keymap = new JSONClass();
+            keymap.Add("template", new JSONData(keyTemplate));
+            installationMetadata.Add("keys", keymap);
```

Comment:
Making the local json file match the shape of the API response (i.e. a nested `keys` object).